ca/django_ca/ca_settings.py

```diff
@@ -329,13 +329,17 @@ def _get_hash_algorithm(setting: str, default: "HashAlgorithms") -> "AllowedHash
     raise ImproperlyConfigured(f"CA_DEFAULT_KEY_SIZE cannot be lower then {CA_MIN_KEY_SIZE}")
 
 
-# CA_DEFAULT_ECC_CURVE can be removed in django-ca==1.25.0
 _CA_DEFAULT_ELLIPTIC_CURVE = getattr(settings, "CA_DEFAULT_ELLIPTIC_CURVE", ec.SECP256R1.name)
 try:
     CA_DEFAULT_ELLIPTIC_CURVE = constants.ELLIPTIC_CURVE_TYPES[_CA_DEFAULT_ELLIPTIC_CURVE]
 except KeyError as ex:
     raise ImproperlyConfigured(f"{_CA_DEFAULT_ELLIPTIC_CURVE}: Unknown CA_DEFAULT_ELLIPTIC_CURVE.") from ex
 
+CA_KEY_BACKENDS: Tuple[str] = tuple(
```

Comment:
Maybe a comment explaining what to set here? The different options/values.

Reply:
will be done in the docs